projects/laji/src/app/shared-modules/spreadsheet/importer/status-cell/error-list/error-list.component.ts

```diff
@@ -65,4 +79,24 @@ export class ErrorListComponent {
     return [value];
   }
 
+  private getMessage(err: unknown): string {
+    if (typeof err !== 'object' || !Util.hasOwnProperty(err, 'message') || typeof err.message !== 'string') {
+      return this.translateService.instant('haseka.form.genericError');
+    }
+    let base = err.message;
+    if (Util.hasOwnProperty(err, 'params') && typeof err.params === 'object') {
```

Comment:
`typeof err.params === 'object'` is `true` for an `Array` and a `null`. Are those possible types for the `err.params` in this problem domain? If not, no need for changes.

(We could add an `isObject` check to the utils that would check that it's not an array or a `null` (maybe just import that from laji-map or laji-form which have that in the utils))

Reply:
True forgot the add the checks for the err.params